modules/farm_fd2/src/entrypoints/transplanting/lib.submit.unit.cy.js

```diff
@@ -0,0 +1,221 @@
+    expect(result.parents[0].attributes.name).to.equal(
+      form.picked[0].data.date + '_' + form.picked[0].data.crop
+    );
+  });
+
+  it('Check the tray inventory quantity--standard assets', () => {
+    expect(result.trayInventoryQuantities[0].attributes.value.decimal).to.equal(
+      form.picked[0].trays
+    );
+    expect(result.trayInventoryQuantities[0].type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.trayInventoryQuantities[0].attributes.measure).to.equal(
+      'count'
+    );
+    expect(result.trayInventoryQuantities[0].attributes.label).to.equal(
+      'Trays'
+    );
+    expect(
+      result.trayInventoryQuantities[0].attributes.inventory_adjustment
+    ).to.equal('decrement');
+  });
+
+  it('Check the asset--plant', () => {
+    // Check the plant asset.
+    expect(result.transplantingPlantAsset.type).to.equal('asset--plant');
+    expect(result.transplantingPlantAsset.attributes.name).to.equal(
+      form.transplantingDate + '_' + form.cropName
+    );
+
+    expect(result.transplantingPlantAsset.attributes.status).to.equal('active');
+    expect(result.transplantingPlantAsset.attributes.notes.value).to.equal(
+      form.comment
+    );
+
+    expect(
+      result.transplantingPlantAsset.relationships.plant_type[0].type
+    ).to.equal(cropToTerm.get(form.cropName).type);
+
+    expect(
+      result.transplantingPlantAsset.relationships.plant_type[0].id
+    ).to.equal(cropToTerm.get(form.cropName).id);
+  });
+
+  it('Check the bed feet quantity--standard', () => {
+    expect(
+      result.transplantingBedFeetQuantity.attributes.value.decimal
+    ).to.equal(form.bedFeet);
+    expect(result.transplantingBedFeetQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingBedFeetQuantity.attributes.measure).to.equal(
+      'length'
+    );
+    expect(result.transplantingBedFeetQuantity.attributes.label).to.equal(
+      'Bed Feet'
+    );
+  });
+
+  it('Check the rows/bed quantity--standard', () => {
+    expect(
+      result.transplantingRowsPerBedQuantity.attributes.value.decimal
+    ).to.equal(form.rowsPerBed);
+    expect(result.transplantingRowsPerBedQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingRowsPerBedQuantity.attributes.measure).to.equal(
+      'ratio'
+    );
+    expect(result.transplantingRowsPerBedQuantity.attributes.label).to.equal(
+      'Rows/Bed'
+    );
+  });
+
+  it('Check the row feet quantity--standard', () => {
+    expect(
+      result.transplantingRowFeetQuantity.attributes.value.decimal
+    ).to.equal(form.rowsPerBed * form.bedFeet);
+    expect(result.transplantingRowFeetQuantity.type).to.equal(
+      'quantity--standard'
+    );
+    expect(result.transplantingRowFeetQuantity.attributes.measure).to.equal(
+      'length'
+    );
+    expect(result.transplantingRowFeetQuantity.attributes.label).to.equal(
+      'Row Feet'
+    );
+  });
```

Comment:
`transplantingRowFeetQuantity` is an inventory adjustment, so we should check the inventory adjustment type and the associated inventory asset to be sure we are adjusting the inventory of the correct plant asset.